db/structure.sql

```diff
@@ -10718,6 +10794,7 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20240124214436'),
 ('20240222032444'),
 ('20240326135332'),
-('20240429211140');
+('20240429211140'),
+('20240618044707');
```

Comment:
I just want to point out there's a merge conflict with structure.sql which likely has to do with the semicolon here since new migrations have since been committed to main